sigh/lib/sigh/resign.rb

```diff
@@ -121,7 +124,7 @@ def installed_identies
       ids = []
       available.split("\n").each do |current|
         begin
-          (ids << current.match(/.*\"(.*)\"/)[1])
+          (ids << "#{current.match(/[a-zA-Z0-9]{40}/)}#{ID_SEPARATOR}#{current.match(/.*\"(.*)\"/)[1]}")
```

Comment:
Hmm, a local test of this code in `irb` doesn't result in any values collected in `ids`. Can you add a test for this portion of the code so that we can be sure it is working as intended.?

You could preference this [commit](https://github.com/baarde/sigh/commit/75a4d4e7070ab824edcd3f2e8425679c50c3c60d) from a previous PR (which wasn't merged due to time constraints while moving to the monorepo) for an alternative approach to matching on SHA-1 or name, but always passing SHA-1 to resign.